delta/rdf/src/main/scala/ch/epfl/bluebrain/nexus/delta/rdf/syntax/JsonSyntax.scala

```diff
@@ -0,0 +1,29 @@
+package ch.epfl.bluebrain.nexus.delta.rdf.syntax
+
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.JsonLdContext
+import ch.epfl.bluebrain.nexus.delta.rdf.utils.JsonUtils
+import io.circe.Json
+import org.apache.jena.iri.IRI
+
+trait JsonSyntax {
+  implicit final def contextSyntax(json: Json): JsonOps = new JsonOps(json)
+}
+
+final class JsonOps(private val json: Json) extends AnyVal {
+
+  def topContextValue: Option[Json] = JsonLdContext.topContextValue(json)
```

Comment:
if this is the intended use of the api, maybe add some docs?

Reply:
All the docs are in the `JsonLdContext`. Do I have to duplicate the docs, or remove them from the `JsonLdContext` and put them on the syntax. 

Since we always said the syntax should not contain logic, why the docs should be there? (note there aren't docs for syntax usually in cats, etc)